dev/kind/BUILD.bazel

```diff
@@ -1,11 +1,40 @@
 package(default_visibility = ["//visibility:public"])
 
-sh_binary(
+load("//rules/helm:def.bzl", helm_template = "template")
+load("//rules/kubectl:def.bzl", kubectl_apply = "apply", kubectl_delete = "delete")
+load(":def.bzl", "kind_start", "kind_delete")
+
+CLUSTER_NAME = "scf"
+NAMESPACE = "scf"
+
+kind_start(
     name = "start",
-    srcs = ["start.sh"],
+    cluster_name = CLUSTER_NAME,
+)
+
+kind_delete(
+    name = "delete",
+    cluster_name = CLUSTER_NAME,
+)
+
+helm_template(
+    name = "rendered_scf",
+    set_values = {
```

Comment:
All values that go into the helm install should come from `value.yaml` files.
Calculated values should be persisted in a **discoverable** and well-known file.

The goal is to allow maximum flexibility to the user, but not constrain the usage of usual commands (currently, if I wanted to use `helm upgrade`, I'd have to remember to `--set system_domain`.